src/modules/collections/controllers/collections.controller.ts

```diff
@@ -229,10 +229,10 @@ export class CollectionsController {
 
 		// Log event
 		const ieObject = await this.ieObjectsService.findBySchemaIdentifier(
-			objectSchemaIdentifier,
+			[objectSchemaIdentifier],
 			referer,
 			getIpFromRequest(request)
-		);
+		)[0];
```

Comment:
make sure you await the right thing
you want to await the return of the function, not the promise[0]
(await this.ieObjectsService.findBySchemaIdentifier())[0]